generator/README.md

```diff
@@ -1 +1,17 @@
-Tools for generating sample 3D Tiles tilesets.
\ No newline at end of file
+Tools for generating sample 3D Tiles tilesets.
+
+# Make Composite Tile
+
+Creates a composite tile from multiple source tiles.
+
+## Example
+`node ./bin/makeCompositeTile.js batched.b3dm instanced.i3dm points.pnts -o output/composite.cmpt`
```

Comment:
>> Does this work with input wildcards? Not a huge deal, but would be fine if it is trivial to implement.

>It does not, I'll see how hard that is to do.

Actually I just tried it and it does work. Your shell handles the wildcards and passes the resolved arguments.